addons/flannel/0.21.5/install.sh

```diff
@@ -270,9 +270,35 @@ function remove_weave() {
 
     # all nodes
     ip link delete weave
+
     rm -rf /var/lib/weave
     rm -rf /etc/cni/net.d/*weave*
     rm -rf /opt/cni/bin/weave*
+
+    logStep "Verifying Weave removal"
+    # Check if any Weave-related resources are still present
+    weave_resources=$(kubectl get all --all-namespaces | grep weave || true)
```

Comment:
why not just target the weave pods? im concerned that other pods may collide or if i named my node with the name weave then all daemonsets would have weave in the name

```suggestion
    weave_resources=$(kubectl -n kube-system get pods --no-headers -l name=weave-net)
```